modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/frontend/data/set/view/table/BaseCMSTableFDSView.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.site.cms.site.initializer.internal.frontend.data.set.view.table;
+
+import com.liferay.frontend.data.set.constants.FDSTimeZoneBehaviorConstants;
+import com.liferay.frontend.data.set.view.table.BaseTableFDSView;
+import com.liferay.frontend.data.set.view.table.DateTimeFDSTableSchemaField;
+
+/**
+ * @author Bruno Carvalho
+ */
+public abstract class BaseCMSTableFDSView extends BaseTableFDSView {
+
+	public DateTimeFDSTableSchemaField addDateFDSTableSchemaField(
+		String fieldName, String label) {
+
+		DateTimeFDSTableSchemaField dateFDSTableSchemaField =
+			new DateTimeFDSTableSchemaField();
+
+		dateFDSTableSchemaField.setContentRenderer(
+			"dateTime"
+		).setFieldName(
+			fieldName
+		).setLabel(
+			label
+		).setLocalizeLabel(
+			true
+		).setSortable(
+			true
+		);
+
+		dateFDSTableSchemaField.setTimeZoneBehavior(
+			FDSTimeZoneBehaviorConstants.APPLY_THEME_DISPLAY_TIME_ZONE);
+
+		return dateFDSTableSchemaField;
```

Comment:
I don't think this belongs in this class. The real fix should be in the FDSTableSchemaBuilder; as a quick fix, I think it would be better to move this logic into a private util class `FDSTableSchemaFieldUtil` with a `getDateTimeFDSTableSchemaFiel(String name, String label)` method that has this logic inside.

Reply:
Anyway, I see this has been implemented like this in other places, so I'm ok with forwarding it.